README.md

```diff
@@ -26,18 +40,140 @@ A slackbot which reads an excel file from sharepoint and returns a specific colu
 4. Add Bot Token Scopes 
     - chat:write
     - commands
+5. go to Slash commands and press Create New Command
+    - Command name: whatever you want it to be
+    - Fill in the required fields and press Save.
+
 
 ### Code 
-1. Clone 
-2. npm i in the project directory 
-3. create a .env file 
+1. Make a folder or move into a prefered one
```

Comment:
Why have you added these instructions? You get all of the stuff described here when you clone the repository. The instructions should just be 1) clone 2) npm i etc...